lib/school_house_web/views/html_helpers.ex

```diff
@@ -35,12 +35,31 @@ defmodule SchoolHouseWeb.HtmlHelpers do
         {"#", "cursor-not-allowed"}
       end
 
-    link(contents,
-      class: "#{class} #{additional_classes}",
-      to: destination
+    additional_classes = if Lessons.coming_soon?(name), do: additional_classes <> " mr-2", else: additional_classes
+
+    content_tag(
+      :div,
+      [
+        link(contents,
+          class: "#{class} #{additional_classes}",
+          to: destination
+        ),
+        Lessons.coming_soon?(name) |> maybe_coming_soon_badge()
+      ],
+      class: "flex flex-wrap"
     )
   end
 
+  def maybe_coming_soon_badge(true),
+    do:
+      content_tag(
+        :span,
+        Gettext.gettext(SchoolHouseWeb.Gettext, "Coming Soon"),
+        class: "rounded py-px px-1 bg-purple text-xs font-semibold self-center flex-shrink-0"
```

Comment:
```suggestion
        class: "rounded py-px px-1 bg-purple text-xs text-white font-semibold self-center flex-shrink-0"
```

I just checked this in light mode and noticed that it uses the default text color which makes it hard to read the badge. This suggestion should resolve this!

<img width="812" alt="Screen Shot 2021-10-05 at 8 23 20 PM" src="https://user-images.githubusercontent.com/1526888/136132390-8d54b5a7-caaa-4333-87c7-246d1064021f.png">

Reply:
Good catch, I didn't check light mode.